### ISSUE TITLE
consul 0.19.3 config

### ISSUE THREAD

**Issue by shatei** [claranet/puppet-consul_template#103]
Hi there.
Tried to deploy consul-template v 0.19.3 with this package but it seems that config now accepts different mapstructure.

```2017/10/01 16:45:50 [WARN] consul now accepts a stanza instead of a string. Update your configuration files and change consul = "" to consul { } instead.
2017/10/01 16:45:50 [ERR] (cli) from file: /etc/consul-template/config/config.json: mapstructure decode failed: 1 error(s) decoding:

* '' has invalid keys: retry, token
```
Is it possible to create new tag so there could be a safe version to checkout branch other than master for the change?

**Comment by sts** [comment]
@shatei should be fixed once #100 gets merged.

**Comment by craigwatson** [comment]
Confirmed that this is fixed in the current master branch, thanks to #100 - closing the issue for now, @shatei please do feel free to re-open if this is still a problem for you :)
